dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1741,8 +1753,26 @@ def _update_helper(self, df_series_clean: pd.Series, profile: dict) -> None:
         :type profile: dict
         :return: None
         """
-        if df_series_clean.empty:
-            return
+        self._greater_than_64_bit = (
```

Comment:
what does `_greater_than_64_bit` mean? If it's true, what should that tell me? If it's false?

Reply:
Oh does `_greater_than_64_bit` mean that it's been converted to numpy if it's true, and it is still pandas if it's false?